actions/testmo-run-submit-thread/action.yml

```diff
@@ -40,14 +40,20 @@ runs:
         ## submit results?
         SUCCESS=0
         if [ ${REPORT} -eq 1 ]; then
+          # red x
```

Comment:
why are we repeating this here? is there a way we could use a script?

Reply:
yeah good question, I tried to put it in a script in nm-actions but since this action was called remotely it cannot find the path of the script to run it, maybe we should put the script in the calling repo, let me try it.

Reply:
updated, looks working, thanks!